### DISCUSSION TITLE
https://www.spsanderson.com/steveondata/posts/2024-08-23/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#6] [Comments]
# https://www.spsanderson.com/steveondata/posts/2024-08-23/

https://www.spsanderson.com/steveondata/posts/2024-08-23/

**Comment by dertuxmalwieder** [comment]
Corrections:
* "The Linux command shell, also known as the terminal" is wrong. A terminal - actually, these days, it usually is a *terminal emulator* like Xterm, Terminator or whatever - *runs* a command shell.
* "Bash is an enhanced version of the original Unix shell program (sh)" is only partially true. The original Unix shell (there were several of them) was quite a different thing. The `bash` - hence its name - was inspired by the Bourne shell, named after its developer, which was introduced in 1979 when the 7th version of UNIX came out. The six versions before had other shells.
* Not all shells support job control.
* FAQ no. 2: No, you can **not** use "the Linux shell" on macOS. You can use a shell that also has a Linux version on macOS, but macOS can't run Linux applications. For example, `bash` on macOS is a macOS shell, not a Linux shell.

**Reply by spsanderson** [reply]
Thank you for posting this. I will amend my post to point to this link for Corrections.

**Reply by spsanderson** [reply]
updated and pushed.
